pkg/vpcmodel/explainabilityPrint.go

```diff
@@ -114,16 +114,22 @@ func (g *groupedConnLine) explainabilityLineStr(c *VPCConfig, connQuery *connect
 	expDetails := g.commonProperties.expDetails
 	filtersRelevant := g.commonProperties.expDetails.filtersRelevant
 	src, dst := g.src, g.dst
+	loadBalancerRule := g.commonProperties.expDetails.loadBalancerRule
 	needEgress := !src.IsExternal()
 	needIngress := !dst.IsExternal()
+	loadBalancerBlocking := loadBalancerRule != nil && loadBalancerRule.Deny()
 	ingressBlocking := !expDetails.ingressEnabled && needIngress
 	egressBlocking := !expDetails.egressEnabled && needEgress
-	var externalRouterHeader, crossRouterFilterHeader, resourceEffectHeader,
-		crossRouterFilterDetails, details string
+	var externalRouterHeader, crossRouterFilterHeader, loadBalancerHeader, resourceEffectHeader,
+		crossRouterFilterDetails, loadBalancerDetails, details string
 	externalRouter, crossVpcRouter, crossVpcRules := expDetails.externalRouter, expDetails.crossVpcRouter, expDetails.crossVpcRules
 	if externalRouter != nil && (src.IsExternal() || dst.IsExternal()) {
 		externalRouterHeader = "External traffic via " + externalRouter.Kind() + ": " + externalRouter.Name() + newLine
 	}
+	if loadBalancerRule != nil {
```

Comment:
In the other cases the header prints only whether the filter/cross vpc router enables the connection or blocks it and the details prints the details of the specific rule in the table/prefix filter that effects the connection. These details includes the index of the rule and e.g. its target, direction (inbound/outbound) etc.

However, here we don't really have such a rule. And the only relevant details are already contained in the header. So I'm not sure this separation is required here. The same text can be used for both

Reply:
fixed